source/includes/examples/cli-example-auth-oidc-stagingprod.rst

```diff
@@ -0,0 +1,16 @@
+.. code-block:: 
+   :copyable: true 
+
+   atlas federatedAuthentication federationSettings identityProvider create oidc IDPName \ 
+     --audience "api://12345678-1234-1234-1234-123456789abc" \ 
+     --authorizationType "GROUP" \
```

Comment:
according to https://www.mongodb.com/docs/atlas/cli/current/command/atlas-federatedAuthentication-federationSettings-identityProvider-create-oidc/ `--authorizationType` must be one of `NONE`, `JWT`, or `SAML`

Reply:
I think the CLI documentation is incorrect. In the example on the [CLI](https://www.mongodb.com/docs/atlas/cli/current/command/atlas-federatedAuthentication-federationSettings-identityProvider-create-oidc/) page, the value is set to `GROUP`. 


```
# Create an identity provider with name IDPName and from your federation settings with federationSettingsId 5d1113b25a115342acc2d1aa.
--
atlas federatedAuthentication federationSettings identityProvider create oidc IDPName --audience "audience" --authorizationType "GROUP" --clientId clientId --desc "IDPName test" --federationSettingsId "5d1113b25a115342acc2d1aa" --groupsClaim "groups" --idpType "WORKLOAD" --issuerUri uri" --userClaim "user"  --associatedDomain "domain"
```

I see in the API [docs](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Federated-Authentication/operation/createIdentityProvider) also that it accepts GROUP or USER only.

Reply:
@kanchana-mongodb let's make sure to capture a DOCSP ticket

Reply:
@fmenezes see https://jira.mongodb.org/browse/DOCSP-45105